Sat Aug 13 16:04:09 CEST 2005  Henning Guenther <der_eq@freenet.de>
  * Substitute @prefix@ with ${PREFIX} to make it user-changeable
  This allows using "make PREFIX=... install". It's usefull if you want to install to a temporary path before merging it into the real path. This makes it possible for gentoo to use yi.
--- old-yi/yi/mk/config.mk.in	2005-08-14 14:40:40.000000000 +0200
+++ new-yi/yi/mk/config.mk.in	2005-08-14 14:40:40.000000000 +0200
@@ -3,10 +3,10 @@
 #
 
 PREFIX=         @prefix@
-BINDIR=         @prefix@/bin
-LIBDIR=         @prefix@/lib/$(PKG)
-DATADIR=        @prefix@/share/doc/$(PKG)
-MANDIR=         @prefix@/man
+BINDIR=         ${PREFIX}/bin
+LIBDIR=         ${PREFIX}/lib/$(PKG)
+DATADIR=        ${PREFIX}/share/doc/$(PKG)
+MANDIR=         ${PREFIX}/man
 IFACEDIR=       $(LIBDIR)/imports
 
 GHC=            @WithGhc@

